src/examples/menu/Nested.tsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useCallback, useState } from 'react';
+import styled from 'styled-components';
+import { hideVisually } from 'polished';
+import { Col, Row } from '@zendeskgarden/react-grid';
+import { Menu, Item, IItemProps } from '@zendeskgarden/react-dropdowns.next';
+import { Separator } from '@zendeskgarden/react-dropdowns';
+
+export const BASE_ITEMS: IItemProps[] = [
+  { value: 'Orange' },
+  { value: 'Berry', type: 'next' },
+  { value: 'Apple' }
+];
+
+export const NESTED_ITEMS: IItemProps[] = [
+  { value: 'Fruits', type: 'previous' },
+  { value: 'separator' },
+  { value: 'Strawberry' },
+  { value: 'Loganberry' },
+  { value: 'Boysenberry' }
+];
+
+const StyledHiddenText = styled.span(hideVisually);
+
+const Example = () => {
+  const [items, setItems] = useState(BASE_ITEMS);
+
+  const handleChange = useCallback(({ type, isExpanded }) => {
+    const isNext = type.includes('next');
+    const isPrev = type.includes('previous');
+
+    if (isNext || isPrev) {
+      setItems(isNext ? NESTED_ITEMS : BASE_ITEMS);
+    } else if (isExpanded === false) {
+      setItems(BASE_ITEMS);
+    }
+  }, []);
+
+  return (
+    <Row justifyContent="center">
+      <Col textAlign="center">
+        <Menu button="Menu" onChange={handleChange}>
```

Comment:
I'm noticing interesting (bug?) mousing behavior where clicking on "Berry" and subsequently hovering over "Strawberry" doesn't change the styling until a different item is hovered. Something to consider for the `Menu` or container?

Reply:
Small update, looks like the behavior is correct in `containers` but broken in `components`. Woohoo? 🤦🏻